src/backend/marsha/core/models/video.py

```diff
@@ -388,3 +388,71 @@ class Meta:
         verbose_name = _("signs language track")
         verbose_name_plural = _("signs language tracks")
         indexes = [NonDeletedUniqueIndex(["video", "language"])]
+
+
+class Thumbnail(BaseModel):
+    """Thumbnail model illustrating a video."""
+
+    video = models.OneToOneField(
```

Comment:
I think this field should be on the video so that the image field is generic and can be used for something else. It could be called `thumbnail`.